google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStubSettings.java

```diff
@@ -100,6 +100,7 @@ public class EnhancedBigtableStubSettings extends StubSettings<EnhancedBigtableS
   private static final int MAX_MESSAGE_SIZE = 256 * 1024 * 1024;
   private static final String SERVER_DEFAULT_APP_PROFILE_ID = "";
 
+  private static final String CBT_ENABLE_DIRECTPATH = "CBT_ENABLE_DIRECTPATH";
```

Comment:
Please add a TODO to document this

Also, if this is will be canonical way to enable DirectPath please update integration tests to use this mechanism:
- update directpath test profiles in the pom to set this env var
  - https://github.com/googleapis/java-bigtable/blob/e7f03fc7d252a7ff6c76a8e6e0a9e6ad3dcbd9d5/google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/test_helpers/env/CloudEnv.java#L208-L210
  -  https://github.com/googleapis/java-bigtable/blob/main/google-cloud-bigtable/pom.xml#L554-L590  
- remove the manual way to enable it: https://github.com/googleapis/java-bigtable/blob/e7f03fc7d252a7ff6c76a8e6e0a9e6ad3dcbd9d5/google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/test_helpers/env/CloudEnv.java#L208-L210

Reply:
done

Reply:
can you tag the todo with your github handle?

Reply:
done